lib/web_of_science/links_client.rb

```diff
@@ -0,0 +1,85 @@
+module WebOfScience
+  # Links AMR (Article Match Retrieval) Service
+  # @see http://ipscience-help.thomsonreuters.com/LAMRService/WebServicesOverviewGroup/overview.html Service documentation
+  class LinksClient
+
+    LINKS_HOST = 'https://ws.isiknowledge.com'.freeze
+    LINKS_PATH = '/cps/xrpc'.freeze
+
+    attr_reader :username, :password, :host
+
+    # @param [String] username
+    # @param [String] password
+    # @param [String] host
+    def initialize(username: nil, password: nil, host: LINKS_HOST)
+      @host = host
+      @username = username
+      @password = password
+      (@username, @password) = Base64.decode64(Settings.WOS.AUTH_CODE).split(':', 2) unless username
+    end
+
+    # Retrieve identifier 'fields' for the record 'ids'
+    # @param [Array<String>] ids
+    # @param [Array<String>] fields (defaults to ['ut', 'doi', 'pmid'])
+    # @return [Hash<String => Hash>]
+    def links(ids, fields = %w(ut doi pmid))
```

Comment:
Would you consider using keyword arguments here, or are you concerned with backwards compatibility?

Reply:
I'm open to using keyword args, although it might require additional changes to any existing consumers.  There should be very few at this stage, it's a new class in this app.

Reply:
OK, done.